.github/workflows/ci.yml

```diff
@@ -33,20 +33,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: dafnik/setup-node-pnpm@v1
```

Comment:
Why do we need a custom actions library here? I had pnpm working in the other PR with the standard package.

Reply:
Yeah, you are right. I was just testing a few things and forgot to revert it back.